src/watchman/models.clj

```diff
@@ -206,12 +218,36 @@
     (k/set-fields fields)
     (k/where {:id id})))
 
+(defn snooze-msg [snooze-until]
+  "Given a Joda datetime, returns a formatted message for the UI indicating until when
+  this role is snoozed"
+  (let [formatter (time-format/formatters :rfc822)]
```

Comment:
Minor, but this is simple enough that it could easily be inlined into the threading expression.